lib/puppet/provider/zfs/zfs.rb

```diff
@@ -58,7 +58,7 @@ def container_property
   # shareiscsi (added in build 120) was removed from S11 build 136
   # aclmode was removed from S11 in build 139 but it may have been added back
   # http://webcache.googleusercontent.com/search?q=cache:-p74K0DVsdwJ:developers.slashdot.org/story/11/11/09/2343258/solaris-11-released+&cd=13
```

Comment:
Could you add a comment here (and a note in your commit message) explaining why we are putting `acltype` here instead of where you originally put it?